__tests__/utils.js

```diff
@@ -32,6 +36,60 @@ test('finds deployment id', t => {
   t.true(id === t.context.plugin.getApiGatewayDeploymentId());
 });
 
+test('finds stage name from deployment', t => {
+  const stageName = {
+    value: 'dev',
+    from: {
+      type: 'AWS::ApiGateway::Deployment',
+      key: 'ApiGatewayDeployment12345'
+    }
+  };
+
+  t.context.template.Resources['ApiGatewayDeployment12345'] = {
+    Type: 'AWS::ApiGateway::Deployment',
+    Properties: {
+      StageName: 'dev'
+    }
+  };
+
+  const realStageName = t.context.plugin.getApiGatewayStageName();
+  t.true(stageName.value === realStageName.value);
+  t.true(stageName.from.type === realStageName.from.type);
+  t.true(stageName.from.key === realStageName.from.key);
+});
+
+test('finds stage name from stage', t => {
+  const stageName = {
+    value: 'attributes_dev',
```

Comment:
can we call this `foo_dev` instead? I realize it's pedantic, but it can help people understand that it's a placeholder/not important detail (also attributes is a Brightspace service, not really relevant to this open source module)

Reply:
Sure